sql/derived_tables/po_lines_eresource.sql

```diff
@@ -5,7 +5,6 @@ CREATE TABLE po_lines_eresource AS
 WITH temp_eresource AS (
```

Comment:
According to the new directive, the documentation line should be at the beginning. Please modify.

Reply:
Have changed that.